### DISCUSSION TITLE
Having Trouble Running a 1.12.2 Forge Server (Possibly due to custom server properties?)

### DISCUSSION THREAD

**Discussion by CaptTracker** [itzg/docker-minecraft-server#2467] [Q&A]
Trying to run a 1.12.2 forge server.
[crash-2023-11-06_03.22.45-server.txt](https://github.com/itzg/docker-minecraft-server/files/13262161/crash-2023-11-06_03.22.45-server.txt)
[docker-compose.txt](https://github.com/itzg/docker-minecraft-server/files/13262166/docker-compose.txt)
Have it set to skip setting up server properties but hasn't fixed anything.  It was running fine earlier idk what happened to it?  Looking for any advice really.

**Comment by itzg** [comment]
> Possibly due to custom server properties?

Can you clarify what you mean?

If you're getting a crash report that is beyond the scope of this image. You may want to ask around on Forge support forums.

**Reply by CaptTracker** [reply]
I changed the server properties manually after installing the forge image.  Stopped working after.  Changed it back still wouldn't work.  Reinstalled it and it worked. Changed the server properties and it didn't.

**Reply by itzg** [reply]
What properties did you change? I'm hoping with that detail someone else can arrive here and provide helpful advice.

**Reply by CaptTracker** [reply]
Few things like seed world type and difficulty.  I was running a docker container like this a few months ago but rebuilt my PC and had to start from scratch.  Not sure if the recent images had an affect on it or not but it sure as hell ain't working now. I can provide the server properties file later if needed?
